README.md

```diff
@@ -1,86 +1,82 @@
-# friByte.no - Zola
+# friByte.no
 
-friByte sin nettside, laget med Zola.
+Made using [Zola][zola].
 
-## Quick start
+## Usage
 
-```
-zola serve # zola serve -O for å åpne direkte i nettleseren
-```
-
-## Oppsett
-
-Installer [Zola](https://getzola.org)
-
-## Deploy
+1. Clone the repository, `git clone git@github.com:fribyte-code/fribyte.no.git`.
+2. Run `zola serve` to run Zola locally.
 
-Når man dytter til `main` så deployes dette live på
-[fribyte.no](https://fribyte.no). Nettsiden bor på VM'en **nettsider** under
-Konrad.
+## Deployment
 
-## Bruk
+- The site is self-hosted on friBytes serveres. It's running Pengebingen.
```

Comment:
Suggestion:
The site is self-hosted on friByte's servers (It's running on Pengebingen).

Reply:
```suggestion
- The site is self-hosted on friByte'ss serveres (It's running Pengebingen).
```

Test out the "Add a suggestion" feature @lucafossen 

![image](https://user-images.githubusercontent.com/5204006/173564408-a8fd9f52-532f-41a8-81fa-94866c4db5e4.png)

Then you will get these nice suggestions that can be automatically committed:

![image](https://user-images.githubusercontent.com/5204006/173564494-33c498b0-aeaa-4fd0-aae7-78d73ccfedc4.png)


And soon be pro in Code review 😎

Reply:
Ah, nice! I did this on the iOS application, not sure if the button exists there..?
will get on desktop asap and try it out